src/ic-cdk-macros/src/lib.rs

```diff
@@ -72,6 +71,30 @@ where
     result.map_or_else(|e| e.to_compile_error().into(), Into::into)
 }
 
+#[allow(missing_docs)]
```

Comment:
Should these two macros have docs?  The macro they are replacing (IIUC) has docs.